### PR TITLE
Django 4.2 migration

### PR DESCRIPTION
**Pull request by etiaro**
Update of Django and other packages

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -20,56 +20,187 @@
...
+"django-appconf==1.0.5"
+"django-compressor==4.3.1"
+"django-debug-toolbar==4.1.0"
+"django-extensions==3.2.3"
+"django-formtools==2.4.1"
+"django-gravatar2==1.4.4"
+"django-js-asset==2.0.0"
+"django-libsass==0.9"
+"django-mptt==0.14.0"
+"django-nested-admin==4.0.2"
```
Comment (A-dead-pixel) [inline setup.py:57]: I don't think this is right (pasted by mistake?):
- no commas
- there are two ``requirements``
- indirect dependencies are included

File: setup.py [hunk 2]
```diff
@@ -20,56 +20,187 @@
...
+    "Django==4.2.2",
+    "pytz==2023.3",
+    "SQLAlchemy==2.0.15",
+    "beautifulsoup4==4.12.2",
+    "PyYAML==6.0",
+    "python-dateutil==2.8.2",
+    "django-two-factor-auth==1.15.2",
+    "django-formtools==2.4.1",
+    "django-registration-redux==2.12",
+    "Celery==4.4.7",    # 5.0 is breaking
```
Comment (A-dead-pixel) [inline setup.py:165]: Can we please freeze just to major versions?
Exact constraints are really troublesome for some (exotic) deployments.

**Review by A-dead-pixel — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (etiaro) [reply setup.py]: Yep, confirmed the fragment was mistaken

**Review by A-dead-pixel — CHANGES_REQUESTED** [review]

**Comment by etiaro** [comment]
@A-dead-pixel okay, the versions are now relaxed as you suggested

**Review by A-dead-pixel — APPROVED** [review]

**Review by twalen — APPROVED** [review]

**Pull request merged**